prow/tide/status.go

```diff
@@ -278,7 +280,7 @@ func (sc *statusController) expectedStatus(log *logrus.Entry, queryMap *config.Q
 
 	repo := config.OrgRepo{Org: crc.Org, Repo: crc.Repo}
 
-	if reason, err := sc.mergeChecker.isAllowed(crc); err != nil {
+	if reason, err := sc.mergeChecker.isAllowedToMerge(crc); err != nil {
```

Comment:
Shouldn't we remove the merge checker from the status controller and use the one from the provider?